src/sentry/incidents/logic.py

```diff
@@ -1205,7 +1229,7 @@ def get_alert_rule_trigger_action_slack_channel_id(name, organization, integrati
     return channel_id
 
 
-def get_alert_rule_trigger_action_msteams_channel_id(name, organization, integration_id):
+def get_alert_rule_trigger_action_msteams_channel_id(name, organization, integration_id, **kwargs):
```

Comment:
It might be better to just add `use_async_lookup` here so that we're at least specific about what kwargs are accepted, rather than accepting anything.